app/models/manageiq/providers/ansible_playbook_workflow.rb

```diff
@@ -16,6 +16,12 @@ def launch_runner
     Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, kwargs)
   end
 
+  def process_abort(*args)
+    response = Ansible::Runner::ResponseAsync.load(context[:ansible_runner_response])
```

Comment:
I think there needs to be more checks here in case we haven't yet created the ansible_runner_response, because the abort happens too early.  In that case, this could be nil (not sure if there could be anything else here, like an unstarted runner).